### PR TITLE
Updating instructions to new EAC

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2582
@dariomws Please kindly review

### CODE REVIEW

**Review by dariomws — APPROVED** [review]
Do we have a confirmation to update all the articles with the steps for the new EAC?

**Comment by get-itips** [comment]
> Do we have a confirmation to update all the articles with the steps for the new EAC?

Trying to get it... @scanum can you help thanks?

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/collaboration-exo/shared-mailboxes.md [hunk 1]
```diff
@@ -38,14 +38,19 @@ You need to be assigned permissions before you can perform this procedure or pro
...
 
-   - **Display name**
+   - **Name**
    - **Email address**
+   - **Alias**
 
-4. To grant Full Access or Send As permissions, click **Add** ![Add Icon](../media/ITPro_EAC_AddIcon.gif), and then select the users you want to grant permissions to. You can use the CTRL key to select multiple users. Confused about which permission to use? See [Which permissions should you use?](#which-permissions-should-you-use) later in this topic.
+4. Click **Create** to save your changes and create the shared mailbox.
+
+5. Under **Next steps** section, Click the **Add users to this mailbox** link.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/shared-mailboxes.md:51]:
```suggestion
5. Under the **Next steps** section, click the **Add users to this mailbox** link.
```

File: Exchange/ExchangeOnline/collaboration-exo/shared-mailboxes.md [hunk 2]
```diff
@@ -38,14 +38,19 @@ You need to be assigned permissions before you can perform this procedure or pro
...
+
+6. To grant Full Access or Send As permissions, click **Add users** button, and then select or search the users you want to grant permissions to. Confused about which permission to use? See [Which permissions should you use?](#which-permissions-should-you-use) later in this topic.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/shared-mailboxes.md:53]:
```suggestion
6. To grant Full Access or Send As permissions, click the **Add users** button, and then select or search the users you want to grant permissions to. Confused about which permission to use? See [Which permissions should you use?](#which-permissions-should-you-use) later in this topic.
```

File: Exchange/ExchangeOnline/collaboration-exo/shared-mailboxes.md [hunk 3]
```diff
@@ -54,17 +59,19 @@ You need to be assigned permissions before you can perform this procedure or pro
 
 ### Use the EAC to edit shared mailbox delegation
 
-1. In the EAC, go to **Recipients** \> **Shared**. Select the shared mailbox, and then click **Edit** ![Edit icon](../media/ITPro_EAC_EditIcon.gif).
+1. In the EAC, go to **Recipients** \> **Mailboxes**. Select the shared mailbox, and then click **Manage mailbox delegation** ![Delegation icon](../media/new-eac-shared-mailbox-delegation.png).
 
-2. Click **Mailbox delegation**.
+2. To grant or remove Full Access (Read and manage) and Send As permissions, click **Edit** next to the permision type.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/shared-mailboxes.md:64]:
```suggestion
2. To grant or remove Full Access (Read and manage) and Send As permissions, click **Edit** next to the permission type.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**